conf/solver/dds_euler.yaml

```diff
@@ -0,0 +1,17 @@
+# @package _global_
+defaults:
+  - oc_base
+  - /prior: gauss_truncate
+  - /sde: vp_10
+  - /model@generative_ctrl: score
+  - /loss: reference_sde_lv_traj
+  - _self_
+
+# Solver
+solver:
+  _target_: sde_sampler.solver.oc.EulerDDS
+
+# Model
```

Comment:
The next three lines should be removed, since `oc_base` has no clipping scheduler.